### PR TITLE
Update quote cart embark routes in NO

### PR DESCRIPTION
**Pull request by robinandeer**
## What?

Add routes for NO home flow and NO home + travel flow with Quote Cart.

## Why?

So we can test them.

**Ticket(s): []**

### CODE REVIEW

File: src/routes.tsx [hunk 1]
```diff
@@ -332,7 +335,26 @@ export const routes: Route[] = [
...
+                case 'home-travel':
+                  if (
+                    window.hedvigClientConfig.appEnvironment === 'production'
+                  ) {
+                    return {
+                      redirect: `/${locale}/new-member`,
+                    }
+                  }
+                  return {
+                    baseUrl: `/${locale}/new-member/home`,
```
Comment (styrelius) [inline src/routes.tsx:353]: Shouldn't this be `/${locale}/new-member/home-travel`? 🤔

File: src/routes.tsx [hunk 2]
```diff
@@ -308,15 +311,15 @@ export const routes: Route[] = [
               switch (name) {
                 case 'contents':
                   return {
-                    baseUrl: `/${locale}/new-member/contents`,
+                    baseUrl: `/${locale}/new-member/${name}`,
```
Comment (styrelius) [inline src/routes.tsx:314]: Smart 🧠

**Review by styrelius — APPROVED** [review]
🇳🇴

File: src/routes.tsx (hunk 2 above)
Comment (robinandeer) [reply src/routes.tsx:314]: hehe thanks, I can open another PR and do the same for the other flows :)

**Pull request merged**